.github/dependabot.yml

```
@@ -46,6 +46,12 @@ updates:
           - "*"
     ignore:
       - dependency-name: "typescript"
+      - dependency-name: "@angular/common"
+        versions: [ ">=22.0.0" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)